blocks/library/gallery/editor.scss

```diff
@@ -13,6 +12,26 @@
 	&.is-transient img {
 		@include loading_fade;
 	}
+
+	.blocks-rich-text__inline-toolbar {
+		margin-top: -1.8em;
```

Comment:
Why change this?

Reply:
Without this change, the inline toolbar is very close to the caption text overlapping part of it this creates a space so things look better.

Reply:
It doesn't look like the rule is applied though. See above.